app/assets/stylesheets/application.scss

```diff
@@ -9,7 +9,17 @@
  * compiled file so the styles you add here take precedence over styles defined in any styles
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
- *
- *= require_tree .
- *= require_self
```

Comment:
Bootstrap Sass recommends to remove require statements and replace with @import's

[twbs/bootstrap-sass](https://github.com/twbs/bootstrap-sass#a-ruby-on-rails)
> Then, remove all the *= require_self and *= require_tree . statements from the sass file. Instead, use @import to import Sass files. Do not use *= require in Sass or your other stylesheets will not be able to access the Bootstrap mixins or variables.